### PR TITLE
update text and links from nwb-conversion-tools to neuroconv

### LINKED ISSUE #63: update links to neuroconv

### PR DESCRIPTION
**Pull request by bendichter**
fix #63

### CODE REVIEW

File: docs/source/conversion_tutorial/03_format_conversions.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+and [``NWBConverter``](https://neuroconv.readthedocs.io/en/main/user_guide/nwbconverter.html). 
 
 ``DataInterface`` provides a unified API for converting data from
 any one data format, such as raw data from an acquisition system or processed
 data from spike sorting or image segmentation software. ``DataInterface``s load the
-raw data in an efficient way, extract metadata, and contain logic for how to write
-this information properly into NWB. ``DataInterfaces`` cover a [large variety of
-different source formats](https://nwb-conversion-tools.readthedocs.io/en/main/api/interfaces.html).
+raw data in an efficient way, extract metadata, and write  this information properly 
+into NWB. ``DataInterfaces``cover a
```
Comment (rly) [inline docs/source/conversion_tutorial/03_format_conversions.md:14]:
```suggestion
into NWB. ``DataInterfaces`` cover a 
```

File: docs/source/conf_extlinks.py [hunk 2]
```diff
@@ -57,7 +57,7 @@
     'hdmf': ('https://hdmf.readthedocs.io/en/stable/', None),
     'pynwb': ('https://pynwb.readthedocs.io/en/stable/', None),
     'pandas': ('https://pandas.pydata.org/pandas-docs/stable/', None),
-    'nct': ('https://nwb-conversion-tools.readthedocs.io/en/main/', None),
+    'neuroconv': ('https://github.com/catalystneuro/neuroconv/en/main/', None),
```
Comment (rly) [inline docs/source/conf_extlinks.py:60]: should this be the readthedocs page instead?

Comment (bendichter) [inline docs/source/conf_extlinks.py:60]:
```suggestion
    'neuroconv': ('https://neuroconv.readthedocs.io/en/main/', None),
```

Comment (bendichter) [inline docs/source/conf_extlinks.py]:
```suggestion
    'neuroconv': ('https://neuroconv.readthedocs.io/en/main/', None),
```

**Review by rly — APPROVED** [review]

**Pull request merged**